src/util.cpp

```diff
@@ -97,15 +97,15 @@ namespace boost {
 
 } // namespace boost
 
-using namespace std;
+
```

Comment:
Because of macros which use the line number, removing this line breaks the same-binaries-check, unfortunately.